client/src/components/FilterOptrions/Filtering/Filtering.jsx

```diff
@@ -2,37 +2,38 @@ import * as actions from "../../../redux/actions";
 import { useSelector } from "react-redux";
 import { useState } from "react";
 import { useDispatch } from "react-redux";
-import { collections } from "../../../collections.json";
 import "./Filtering.css";
 
 export default function Filtering() {
   //aqui van los estados de filtado que se encuentran en redux/reducer.
-  const [selectedCollection, setSelectedCollection] = useState("");
+  const [, setSelectedCollection] = useState("");
   const [selectedCategory, setSelectedCategory] = useState("");
   const [selectedState, setSelectedState] = useState("");
 
   const dispatch = useDispatch();
 
   const nfts = useSelector((state) => state.nfts);
-
   const collections = useSelector((state) => state.collections);
 
   let filterCollection = [], count = 0;
-  collections.map(e => {
+  collections.forEach(e => {
     count = 0;
```

Comment:
para recorrer un array, solo recorrerlo es mejor usar for each... lo demas genera warnings